### PR TITLE
feat: add AI idea generation API route

### PR DESCRIPTION
**Pull request by vpavlin**
This PR adds a simple API route to query Akash Chat API and return new idea generated by `Meta-Llama-3-1-8B-Instruct-FP8` model.

It needs a `CHAT_API_KEY` env var - you can get your API key from https://chatapi.akash.network/

related to #3

### CODE REVIEW

File: pages/api/ai.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import ideas from "../../public/ideas/expert-ideas.json"
+
+const input_json = JSON.stringify(ideas)
+const system_prompt =  `You are human tasked with coming up with privacy focused project ideas for hackathons. Here is a list of example: ${input_json} Do not return any of these example right away, but you are alowed to combine them into new ideas. If you use any of the example as inspiration, add the list in output as 'basedOn'. Only print the result in the same format as the example inputs`
+const url = "https://chatapi.akash.network/api/v1/chat/completions"
```
Comment (0xjei) [inline pages/api/ai.js:5]: we can move this to the env as well

File: pages/api/ai.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+const chatApiKey = process.env.CHAT_API_KEY
+console.log(chatApiKey)
```
Comment (0xjei) [inline pages/api/ai.js:7]: we can get rid of this .log

File: pages/api/ai.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+    if (body.keywords == undefined) {
+        res.status(400).json({error: "missing keywords"})
+        return
+    }
+
+    const main_prompt = `Provide an idea based on keywords: ${body.keywords}; (ignore: ${new Date()})`
+
+    const data = {                                                                                                                                                                                                                                                                                                    
+        model: "Meta-Llama-3-1-8B-Instruct-FP8",
```
Comment (0xjei) [inline pages/api/ai.js:25]: Also this can be moved on the .env

**Review by 0xjei — COMMENTED** [review]
tysm @vpavlin <3 lgtm - just soft approving here to check w/ @Msiusko and @EclecticSamurai if it's what they have in mind!

**Comment by vpavlin** [comment]
@0xjei  Ha, I had these fixed in my local repo, but forgot to push:D Sorry about that:)

**Review by 0xjei — APPROVED** [review]
lgtm! thank you so much @vpavlin

**Pull request merged**